### PR TITLE
Back up your community's private key

### PR DESCRIPTION
**Pull request by jorge-campo**
- 311 First draft, waiting for additional info -- Not ready for review.
- Rebase 'develop' branch - 311 second draft -- ready for review
- 311 update 'import' article title and references
- 311 clarify statements based on 'Import your Status Community' content
- 311 updates after John's review

### CODE REVIEW

**Comment by jorge-campo** [comment]
Hey @cheny0 -- Can you review this article and let me know your thoughts?

Thanks!

File: docs/en/status-communities/back-up-your-community-s-private-key.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+## Common questions
+
+### Where is my community information backed up?
+
+Your community configuration and messages for up to 30 days are cached in the Waku [peer-to-peer network][peer-to-peer-messaging]. The community's private key represents proof of ownership but doesn't store the community configuration and messages.
+
+### How frequently should I back up the community's private key?
+
+You only need to back up your community's private key once. You can create the backup right after you [create the community][create-a-status-community].
```
Comment (cheny0) [inline docs/en/status-communities/back-up-your-community-s-private-key.md:50]:
```suggestion
You only need to back up your community's private key once, as long as you don't lose it. You can create the backup right after you [create the community][create-a-status-community].
```

**Review by cheny0 — COMMENTED** [review]
Hi @jorge-campo , adding my comments here :)

File: docs/en/status-communities/back-up-your-community-s-private-key.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+!!! warning "Caution"
+    Manually transferring your community from one computer to another by copying the private key is an insecure method of handling a private key. Still, it's currently the only way to do so.</br>
+    We're aware of this issue and plan to implement a secure solution in a future release. In the meantime, take extra precautions when handling your community key and keep it safe.
+
+## What to expect
+
+- As a community owner, you're responsible for backing up your community's private key and keeping it safe. Status doesn't have access to this key and can't recover it for you.
+- Anyone with access to your community's private key may become the owner of the community and control its settings, permissions and membership approvals.
+- If no new messages are posted to your community for 30 days or more, all the community messages and settings are irretrievably lost, even if you have the community's private key.
```
[the remark this answers is not in the document]
Comment (jorge-campo) [reply docs/en/status-communities/back-up-your-community-s-private-key.md:26]: Updated; thanks!

**Pull request merged**